todo-app/src/features/toggle-task/ui.tsx

```diff
@@ -1,5 +1,7 @@
 import { Checkbox } from "antd"; // ~ "shared/ui/checkbox"
+
 import { taskModel, taskLib } from "entities/task";
+import { toggleTaskModel } from '.';
```

Comment:
Import `model` directly `from './model'` to prevent dependency issues

Reply:
Done.